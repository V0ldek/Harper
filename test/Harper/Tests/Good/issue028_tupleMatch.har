gcd :: Integer -> Integer -> Integer;
gcd a b = match (a, b) {
  (_, 0) => a,
  _      => gcd b | a mod b
};

main :: Integer;
main = gcd 65482732497 4738130263965527;

