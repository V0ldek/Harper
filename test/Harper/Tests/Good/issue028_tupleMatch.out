Execution ended with value: 1400191